tests/rbd/rbd_utils.py

```diff
@@ -647,8 +648,13 @@ def migration_prepare(self, src_spec, dest_spec):
             src_spec : source image spec SOURCE_POOL_NAME/SOURCE_IMAGE_NAME
             dest_spec: Target image spec TARGET_POOL_NAME/SOURCE_IMAGE_NAME
         """
-        log.info("Starting prepare Live migration of image ")
-        return self.exec_cmd(cmd=f"rbd migration prepare {src_spec} {dest_spec}")
+        log.info("Starting prepare Live migration of image")
+        command = "rbd migration prepare "
+        if ".json" not in src_spec:
+            command += f"{src_spec} {dest_spec}"
+        else:
+            command += f"{dest_spec} --import-only --source-spec-path {src_spec}"
```

Comment:
please add info about --import-only in method doc

Reply:
Added doc about method input